aircraft/f-14b/gui/dialogs/iff-datalink.xml

```diff
@@ -78,7 +78,9 @@
             <halign>right</halign>
             <binding>
                 <command>nasal</command>
-                <script><![CDATA[setprop("instrumentation/datalink/channel", 0);]]></script>
+                <script>
+                    <![CDATA[setprop("instrumentation/datalink/channel", 0);]]>
```

Comment:
although I realise that you didn't author this setting a property using Nasal here isn't the right thing to do because we have a better supported way of doing this in FG.

Instead of Nasal use the binding property-assign - as follows:

```
          <binding>
                <command>property-assign</command>
                <property>instrumentation/datalink/channel</property>
                <value>aalights/num-warns</value>
            </binding>

```